zebra/zapi_msg.c

```diff
@@ -730,16 +731,26 @@ static int route_notify_internal(const struct prefix *p, int type,
 
 	stream_putl(s, table_id);
 
+	/* Encode AFI, SAFI in the message */
+	if (type == ZEBRA_ROUTE_BGP) {
```

Comment:
This is going to create buggy behavior in the future.  Just always send the afi/safi.